cachi2/core/package_managers/gomod.py

```diff
@@ -79,10 +79,146 @@ class GoPackage(_GolangModel):
 
     import_path: str
     standard: bool = False
-    module: Optional[GoModule]
+    module: Optional[ParsedModule]
     deps: list[str] = []
 
 
+class Module(NamedTuple):
```

Comment:
I was a little on the fence with these extra wrapper models, but the code felt better than not having them and going straight to SBOM components. I left the earlier version in a separate commit, so you can compare it.